doc/mgmt/SONiC_OC_Management_VRF_HLD.md

```diff
@@ -0,0 +1,305 @@
+           |  +--rw config
+           |  |  +--rw mac-learning?      boolean
+           |  |  +--rw mac-aging-time?    uint16
+           |  |  +--rw maximum-entries?   uint16
+           |  +--ro state
+           |  |  +--ro mac-learning?      boolean
+           |  |  +--ro mac-aging-time?    uint16
+           |  |  +--ro maximum-entries?   uint16
+           |  +--rw mac-table
+           |     +--rw entries
+           |        +--rw entry* [mac-address vlan]
+           |           +--rw mac-address    -> ../config/mac-address
+           |           +--rw vlan           -> ../config/vlan
+           |           +--rw config
+           |           |  +--rw mac-address?   yang:mac-address
+           |           |  +--rw vlan?          -> ../../../../../../vlans/vlan/config/vlan-id
+           |           +--ro state
+           |           |  +--ro mac-address?   yang:mac-address
+           |           |  +--ro vlan?          -> ../../../../../../vlans/vlan/config/vlan-id
+           |           |  +--ro age?           uint64
+           |           |  +--ro entry-type?    enumeration
+           |           +--rw interface
+           |              +--rw interface-ref
+           |                 +--rw config
+           |                 |  +--rw interface?      -> /oc-if:interfaces/interface/name
+           |                 |  +--rw subinterface?   -> /oc-if:interfaces/interface[oc-if:name=current()/../interface]/subinterfaces/subinterface/index
+           |                 +--ro state
+           |                    +--ro interface?      -> /oc-if:interfaces/interface/name
+           |                    +--ro subinterface?   -> /oc-if:interfaces/interface[oc-if:name=current()/../interface]/subinterfaces/subinterface/index
++          +--rw config
++          |  +--rw name?                       string
++          |  +--rw type?                       identityref
++          |  +--rw enabled?                    boolean
++          |  +--rw description?                string
+           |  +--rw router-id?                  yang:dotted-quad
+           |  +--rw route-distinguisher?        oc-ni-types:route-distinguisher
+           |  +--rw enabled-address-families*   identityref
+           |  +--rw mtu?                        uint16
++          +--ro state
++          |  +--ro name?                       string
++          |  +--ro type?                       identityref
++          |  +--ro enabled?                    boolean
++          |  +--ro description?                string
+           |  +--ro router-id?                  yang:dotted-quad
+           |  +--ro route-distinguisher?        oc-ni-types:route-distinguisher
+           |  +--ro enabled-address-families*   identityref
+           |  +--ro mtu?                        uint16
+           +--rw encapsulation
+
+module: sonic-mgmt-vrf
+    +--rw sonic-mgmt-vrf
+       +--rw MGMT_VRF_CONFIG
+          +--rw MGMT_VRF_CONFIG_LIST* [mgmt-vrf-name]
```

Comment:
No there will be only 1 entry in this table. But CVL mandates the sonic yang to have LIST at this level. Since there will be only 1 key (VRF_GLOBAL) for this table, there will be only 1 entry.